browser-interface/packages/unity-interface/kernelConfigForRenderer.ts

```diff
@@ -17,6 +25,14 @@ export function kernelConfigForRenderer(): KernelConfigForRenderer {
     network = getSelectedNetwork(globalState)
   } catch {}
 
+  const COLLECTIONS_OR_ITEMS_ALLOWED =
```

Comment:
should be required to use the config: `BUILDER_SERVER_URL` to allow fetching the `collections/wearables/emotes` from a `builder-api`?

Reply:
It is not needed because `BUILDER_SERVER_URL` is only used in conjunction with `WITH_FIXED_ITEMS` or `WITH_FIXED_COLLECTIONS`. An in that case we are already setting the `urlParamsForWearablesDebug` to `true`:

```
const urlParamsForWearablesDebug =
    !!(WITH_FIXED_ITEMS || WITH_FIXED_COLLECTIONS || COLLECTIONS_OR_ITEMS_ALLOWED)
```